### PR TITLE
fix: wiki related fixes

### PR DESCRIPTION
**Pull request by JustinCast**
# Wiki Related Fixes

- [ ] Move ipfs pinning to new graphql endpoint (done, partially). Image pinning will be managed in a separated PR ([issue](https://github.com/EveripediaNetwork/issues/issues/229))
- [x] Fix max words length
- [x] Fix tags button disabled/enabled bugs
- [x] Wiki image validation error

## How should this be tested?
- Using the attached link

## Linked issues

closes https://github.com/EveripediaNetwork/issues/issues/217
closes https://github.com/EveripediaNetwork/issues/issues/167

### CODE REVIEW

File: src/components/Layout/Editor/Highlights/HighlightsModal/Tags.tsx [hunk 1]
```diff
@@ -65,8 +54,19 @@ const Tags = () => {
...
   }
 
-  const disableAddTagButton = () =>
-    inputTagRef === null || inputTagRef.current?.value === ''
+  const handleOnTagInputChanges = (value: string) => {
+    if (value === '') setAddBtnDisabled(true)
+    else if (value.indexOf(' ') >= 0) {
+      setTagState({ msg: "Name can't contain blank spaces", invalid: true })
+      setAddBtnDisabled(true)
+    } else if (value.length >= 15) {
```
Comment (kesar) [inline src/components/Layout/Editor/Highlights/HighlightsModal/Tags.tsx:62]: move magic numbers to constant on top of the file

File: src/pages/create-wiki/index.tsx [hunk 2]
```diff
@@ -160,7 +163,12 @@ const CreateWiki = () => {
   const getWikiSlug = () => slugify(String(wiki.title).toLowerCase())
 
   const isValidWiki = () => {
-    if (wiki.images?.length === 0) {
+    if (
+      !image ||
+      image.type === null ||
+      image.type === undefined ||
+      (image.type as ArrayBuffer).byteLength === 0
```
Comment (kesar) [inline src/pages/create-wiki/index.tsx:170]: extract into a small function that explain what you are trying to achieve here

Comment (JustinCast) [reply src/pages/create-wiki/index.tsx:172]: Oh yes 100%, makes sense

File: src/pages/create-wiki/index.tsx [hunk 3]
```diff
@@ -78,6 +81,8 @@ const types = {
   ],
 }
 
+const MINIMUM_WORDS = 300
```
Comment (kesar) [inline src/pages/create-wiki/index.tsx:84]:
```suggestion
const MINIMUM_WORDS = 150
```

**Review by kesar — APPROVED** [review]
looks good let's just commit the words to 150 for now, specially now that we are in testing mode 👍🏻

**Pull request merged**